bf2fc6cc711aee1a0c2a/kas-fault-tests
# Add cloud layer test (disable AZ)

## Description
[MGDSTRM-1581](https://issues.redhat.com/browse/MGDSTRM-1581)

Addition of Kraken cloud-layer test to the fault testing pipeline.

> **_IMPORTANT:_** this PR should be merged until the pipeline is in a working state (https://github.com/bf2fc6cc711aee1a0c2a/kas-fault-tests/pull/13) at which point this PR may need to be rebased. 

## Verification
- The pipeline uses the [aws_mk_running_the_service](https://vault.devshift.net/ui/vault/secrets/managed-services/show/rts/aws_mk_running_the_service) AWS account. Create a cluster in a region **that no other cluster is located**. . You can see what clusters are in what regions by running: `ocm list cluster`, then choose a region that is no being used so that the test does not interfere with other clusters.
- Run the pipeline using the `cloud` level tests parameter on the pipeline. Choose the AWS Region that the cluster is located in via the drop-down menu
- Ensure that the AZ is brought down: the clusters console should be unavailable for a short period of time.
- Ensure that the pipeline completes without error, and contains adequate logging in the pipelines console output.
